src/libraries/Microsoft.PowerFx.Core/Public/Values/ErrorValue.cs

```diff
@@ -68,5 +72,40 @@ private static IEnumerable<ExpressionError> CombineErrors(IEnumerable<ErrorValue
                 }
             }
         }
+
+        public override string ToExpression()
```

Comment:
>    public override string ToExpression() [](http://example.com/codeflow?start=4&length=41)

This is an example of why this overload would be good:
 public override void ToExpression(StringBuilder sb)

And then we can have a convenience method:
public string ToExpression()
{
 var sb = new StringBuilder();
 this.ToExpression(sb);
 return sb.ToString();
}